sklearn/random_projection.py

```diff
@@ -599,6 +716,12 @@ class SparseRandomProjection(BaseRandomProjection):
         If False, the projected data uses a sparse representation if
         the input is sparse.
 
+    fit_inverse_transform : bool, default=False
+        Learn the inverse transform by computing the pseudo-inverse of the
+        components during fit. Note that the pseudo-inverse is a dense array.
```

Comment:
```suggestion
        components during fit. Note that the pseudo-inverse is always a
        dense array even if the training data was sparse. This means that
        it might be necessary to call `inverse_transform` on small batch
        of samples at a time to avoid exhausting the available memory on
        the host.
```